src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1794,7 +1813,7 @@ namespace winrt::TerminalApp::implementation
     {
         for (auto& tab : tabs)
         {
-            co_await _RemoveTab(tab);
+            co_await _HandleCloseTabRequested(tab);
```

Comment:
Is this one correct? This is an explicit action for _removing all the tabs_. What's it hooked up to, and is that thing expecting that it simply request closure?

Reply:
What is the user experience here like? Do we get multiple dialogs? "Yes, close all tabs. YES, I understand this tab is read-only."

Reply:
Yes. This is the behavior we have today, that for each read-only tab asks if it should be closed or not.